### PR TITLE
chore: update datasets

### PR DESCRIPTION
**Pull request by jjfrench**
## Summary:
- removed NASA JPL biomass layer since we've removed the current version from MAAP
- updates datasets in the MAAP STAC JSON

### Fixes or Addresses Issue \#: [ticket number & link]
- https://github.com/NASA-IMPACT/active-maap-sprint/issues/787

## Checklist before requesting a review:
- [ ] My code follows the guidelines of this project
- [x] I  performed a self-review of my code changes
- [X] I have added my changes to the change log (`HISTORY.rst`)
- [ ] I have completed spell-checking, removed unnecessary print statements & commented-out code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by emmalu — APPROVED** [review]
👍🏽

**Comment by wildintellect** [comment]
@jjfrench what about the GEDI version change and the CCI Biomass mosaic regenerated hash?

**Comment by jjfrench** [comment]
@emmalu, should the ESACCI biomass layers have the `?assets=` defined?

**Comment by wildintellect** [comment]
@jjfrench yes when using titiler-pgstac you need the assets, the old url used regular titiler that doesn't use that parameter (since it's not reading the STAC records)

**Comment by jjfrench** [comment]
@wildintellect, should be good to merge then?

File: stac_ipyleaflet/widgets/basemaps.py
Comment (wildintellect) [inline stac_ipyleaflet/widgets/basemaps.py:1]: Interesting we should open a ticket about alternative Basemap options.

**Review by wildintellect — APPROVED** [review]

**Comment by jjfrench** [comment]
https://www.drupal.org/project/leaflet_more_maps/issues/3393412
https://github.com/jupyter-widgets/ipyleaflet/issues/1144
https://stamen.com/stamen-x-stadia-the-end-of-the-road-for-stamens-legacy-map-tiles/

Stamen basemap currently broken until ipyleaflet update

**Comment by jjfrench** [comment]
@wildintellect, I still haven't confirmed that the config loads correctly - debugging issues (the basemap being one)

**Comment by jjfrench** [comment]
image
JPL Dataset Removed

**Pull request merged**